ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -873,6 +882,35 @@ protected void maybeProcessCallbacks() {
       }
     }
 
+    /**
+     * Decompress the compressed buffer specified.
+     * If the buffer is not compressed, based on blobCompressed property, then return the input buffer.
+     *
+     * @param sourceBuffer The source buffer to decompress.
+     * @return New buffer if decompressed successfully and the source buffer is released.
+     *         Original buffer is blob not compressed.  NULL if failed and source buffer is released.
+     */
+    protected ByteBuf decompressContent(ByteBuf sourceBuffer) {
+      if (!isChunkCompressed) {
+        logger.info("Blob is not compressed.  Return source buffer.");
+        return sourceBuffer;
+      }
+
+      // Note: decompress() already emit metrics to for decompress() failure.
+      try {
+        ByteBuf result = decompressionService.decompress(sourceBuffer, routerConfig.routerMaxPutChunkSizeBytes);
+        sourceBuffer.release();
+        return result;
+      }
+      catch (Exception ex) {
```

Comment:
nit: better to use below format to be consistent:
`} catch (...) {`

Reply:
I just searched and fixed all them all in the project.